pkg/cmd/config/clear-cache/clear_cache.go

```diff
@@ -23,13 +22,10 @@ func NewCmdConfigClearCache(f *cmdutil.Factory, runF func(*ClearCacheOptions) er
 	}
 
 	cmd := &cobra.Command{
-		Use:   "clear-cache",
-		Short: "Clear the cli cache",
-		Example: heredoc.Doc(`
-			# Clear the cli cache
-			$ gh config clear-cache
-		`),
-		Args: cobra.ExactArgs(0),
+		Use:     "clear-cache",
+		Short:   "Clear the cli cache",
+		Example: "$ gh config clear-cache",
+		Args:    cobra.ExactArgs(0),
```

Comment:
Aside from restoring the long `Example`, if we're touching this then we should consider adding a `Long` section that explains how this clears out cached API request data and whatever else is included.